snapcraft/tests/test_lifecycle.py

```diff
@@ -678,6 +686,36 @@ def test_prime_with_build_info_records_snapcraft_yaml(self):
 description: test
 confinement: strict
 grade: stable
+
+
```

Comment:
This is because the make_snapcraft_yaml helper leaves room in case you want to set the type. But when you don't set the type, it still leaves the empty line. I could fix it, but that would make the helper more complicated, and this is the only place where we actually check the full contents, so I would prefer not to.